### PR TITLE
Week 9 Documentation Updates

### LINKED ISSUE #51: Week 9: Update UG - Quick start - Update Screenshot

### LINKED ISSUE #60: Week 9: Update UG - Features: Customise Order
1. Update default order.
1. Add another example (1 argument) that shows the usage of the default ordering.

### LINKED ISSUE #66: Week 9: Update UG - Features: Customise Hide/Show
1. Add another example with multiple arguments to show that the commands `customise hide` and `customise show` can take in multiple arguments.
1. Fix the phrasing of the description of the commands `customise hide` and `customise show`.

### PR DESCRIPTION
**Pull request by PangKuangWei**
1. Update documenation for `customise order`, `customise hide` and `customise show`
1. Update Ui.png 
1. Add skeleton implementation details for `customise order` in the developer guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -229,18 +230,20 @@ Format: `customise show [p/] [e/] [a/] [t/]`
...
 
 Format: `customise order [p/] [e/] [a/] [t/]`
 
-*Attributes not given will be ordered last in default order. (tag > phone > address > email)
+* Attributes not given will be ordered last in default order. (tag > phone > email > address)
 
 Example:
 
 * `customise order a/ e/ p/` The application now shows address first, followed by email, phone, then tags.
+* `customise order a/` The application now shows address first the rest of the attributes will follow the default order, therefore address will be followed by tags, phone and then email.
```
Comment (florentianayuwono) [inline docs/UserGuide.md]:
```suggestion
* `customise order a/` The application will show address first. The rest of the attributes will follow the default order. Therefore address will be followed by tags, phone and then email.
```

**Review by florentianayuwono — COMMENTED** [review]
Great work!

File: docs/UserGuide.md (hunk 1 above)
Comment (PangKuangWei) [reply docs/UserGuide.md]: Ok, I have changed it.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -229,18 +230,20 @@ Format: `customise show [p/] [e/] [a/] [t/]`
...
 Format: `customise order [p/] [e/] [a/] [t/]`
 
-*Attributes not given will be ordered last in default order. (tag > phone > address > email)
+* Attributes not given will be ordered last in default order. (tag > phone > email > address)
 
 Example:
 
-* `customise order a/ e/ p/` The application now shows address first, followed by email, phone, then tags.
+* `customise order a/ e/ p/` The application will show address first, followed by email, phone, then tags.
+* `customise order a/` The application will show address first. The rest of the attributes will follow the default order, therefore address will be followed by tags, phone and then email.
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]:
```suggestion
* `customise order a/` The application will show address first. The rest of the attributes will follow the default order, therefore address will be followed by tags, phone number and then email.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -229,18 +230,20 @@ Format: `customise show [p/] [e/] [a/] [t/]`
...
 
 Format: `customise order [p/] [e/] [a/] [t/]`
 
-*Attributes not given will be ordered last in default order. (tag > phone > address > email)
+* Attributes not given will be ordered last in default order. (tag > phone > email > address)
 
 Example:
 
-* `customise order a/ e/ p/` The application now shows address first, followed by email, phone, then tags.
+* `customise order a/ e/ p/` The application will show address first, followed by email, phone, then tags.
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]:
```suggestion
* `customise order a/ e/ p/` The application will show address first, followed by email, phone number, then tags.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -285,7 +288,7 @@ _Details coming soon ..._
 |----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | **Add**        | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague` |
 | **Clear**      | `clear`                                                                                                                                                               |
-| **Customise**  | `customise hide [p/] [e/] [a/] [t/]` <br> `customise show [p/] [e/] [a/] [t/]` <br> `customise order [p/] [e/] [a/] [t/]` <br> e.g, `customise order a/ e/ p/`        |
+| **Customise**  | `customise hide [t/] [p/] [e/] [a/]` <br> `customise show [t/] [p/] [e/] [a/]` <br> `customise order [t/] [p/] [e/] [a/]` <br> e.g, `customise order a/ e/ p/`        |
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]: Add examples for the other 2 subcommands?

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Customise Order Feature
+
+#### Implementation
+
+This feature has 2 crucial methods for this feature are in the `CustomiseCommand` and `PersonCard` classes.
```
Comment (Eugene-Ong-W-X) [inline docs/DeveloperGuide.md]: Better phrasing? `This feature has 2 crucial methods for this feature are in...` is pretty confusing to read

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
...
+
+* `CustomiseCommand#changeAttributeOrder(Model model)` - saves the new attribute order in the `preferences.json`
```
Comment (Eugene-Ong-W-X) [inline docs/DeveloperGuide.md]: `in the preferences.json` can be `in preferences.json` or `in the preferences.json file`

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
...
+* `PersonCard#setAttributes()` - reads the order from the `preferences.json` file and builds the person card in the order specified.
```
Comment (Eugene-Ong-W-X) [inline docs/DeveloperGuide.md]: `person card` should be `PersonCard`

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
...
+
+_{diagrams and more in depth explanation to be added}_
+
+#### Design considerations:
```
Comment (Eugene-Ong-W-X) [inline docs/DeveloperGuide.md]: `Design consideration`, removing the `s:` to make the TOC look nicer

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
...
+* **Alternative 1 (current choice):** Sets the order in 4 placeholder JavaFX flowpanes.
```
Comment (Eugene-Ong-W-X) [inline docs/DeveloperGuide.md]: `flowpanes` can be `FlowPane`

**Review by Eugene-Ong-W-X — COMMENTED** [review]
Some possible changes to consider

File: docs/UserGuide.md (hunk 2 above)
Comment (PangKuangWei) [reply docs/UserGuide.md]: Ok, I have added it.

File: docs/UserGuide.md (hunk 3 above)
Comment (PangKuangWei) [reply docs/UserGuide.md]: Ok, I have added it.

File: docs/UserGuide.md (hunk 4 above)
Comment (PangKuangWei) [reply docs/UserGuide.md]: Okay, I shall add examples for the other two.

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (PangKuangWei) [reply docs/DeveloperGuide.md]: Ok, I have changed it.

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (PangKuangWei) [reply docs/DeveloperGuide.md]: Ok, I have changed it to `in the preferences.json file`.

File: docs/DeveloperGuide.md (hunk 7 above)
Comment (PangKuangWei) [reply docs/DeveloperGuide.md]: Okay I shall change it.

File: docs/DeveloperGuide.md (hunk 8 above)
Comment (PangKuangWei) [reply docs/DeveloperGuide.md]: Ok, I have removed the `s`.

File: docs/DeveloperGuide.md (hunk 9 above)
Comment (PangKuangWei) [reply docs/DeveloperGuide.md]: Ok, I have changed it.

**Review by Eugene-Ong-W-X — APPROVED** [review]
LGTM

**Pull request merged**